src/config.rs

```diff
@@ -38,6 +38,33 @@ impl Default for SortOrder {
     }
 }
 
+#[derive(Debug, Eq, PartialEq, Hash, Copy, Clone)]
+#[derive(Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub enum SassOutputStyle {
+    Nested,
+    Expanded,
+    Compact,
+    Compressed,
+}
+
+#[derive(Debug, PartialEq)]
+#[derive(Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct SassOptions {
+    pub import_dir: String,
```

Comment:
Is there a use case for import_dir to be configurable?  The problem with extra configuration is that it means there is extra places corner cases to handle.  For example, we should probably enforce that this directory starts with a `_` to ensure it doesn't get picked up as assets.

I'm sorry, we talked about what to name this but I lost track of my questioning what should be hard coded or configurable.